dist/tools/openocd/openocd.sh

```diff
@@ -334,6 +383,7 @@ do_debug() {
             ${OPENOCD_DBG_EXTRA_CMD} \
             -c 'targets' \
             ${OPENOCD_DBG_START_CMD} \
+            -c 'targets ${OPENOCD_CORE}' \
```

Comment:
Why is this needed?

Reply:
From commit message:

> For multi-arch SoC like STM32MP1, the right target core has
> to be selected to avoid debugging the wrong default cpu.
> This is done using openocd command 'targets ${OPENOCD_CORE}'.
> OPENOCD_CORE has to be set in board Makefile.include file.
> In case it is not set, the command just display available targets, thus it
> has no effect on already existing boards using openocd.